pulser-core/pulser/json/abstract_repr/schemas/config-schema.json

```diff
@@ -0,0 +1,474 @@
+            "eigenstates",
+            "amplitudes"
+        ]
+    },
+    "operator": {
+        "description": "an operator, stored in a serialized format",
+        "type": "object",
+        "properties": {
+            "eigenstates": {
+                "type": "array",
+                "items": {
+                    "enum": [
+                        "u",
+                        "d",
+                        "r",
+                        "g",
+                        "h",
+                        "x",
+                        "0",
+                        "1"
+                    ]
+                }
+            },
+            "n_qudits": {
+                "type": "integer"
+            },
+            "operations": {
+                "description": "array representing the operator as a weighted sum of tensor product operators",
+                "type": "array",
+                "items": {
+                    "type": "array",
+                    "prefixItems": [
+                        {
+                            "description": "weight for the tensor product operator",
+                            "$ref": "#/number_or_complex"
+                        },
+                        {
+                            "type": "array",
+                            "description": "tensor product operator, represented as a list of single qubit operators and the qubits upon which to apply it. Identity ops are implied",
+                            "items": {
+                                "type": "array",
+                                "description": "tuple of single qudit operator + support",
+                                "prefixItems": [
+                                    {
+                                        "type": "object",
+                                        "description": "single qudit operator, the matrix elements are encoded using a string format",
+                                        "additionalProperties": {
+                                            "description": "value of the matrix element in the key",
+                                            "$ref": "#/number_or_complex"
+                                        }
+                                    },
+                                    {
+                                        "type": "array",
+                                        "description": "set of qudits (represented by their indices), to which apply the defined operator",
+                                        "items": {
+                                            "type": "integer"
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+        "required": [
+            "eigenstates",
+            "n_qudits",
+            "operations"
+        ]
+    },
+    "expectation": {
```

Comment:
Can we make all observables not accept additional properties? I don't see any case where that would make sense 
For that, just set `additionalProperties: false` in each one
 @a-quelle do you have an objection?